src/select.ts

```diff
@@ -179,20 +184,23 @@ export class SelectQuery<
   ): T extends TableDefinition<any> ? never : Join<SelectQuery<Columns, IncludesStar>, T> {
     return this.newSelectQuery(
       [...this.tokens, new StringToken(`FROM`), ...fromItem.toTokens()],
+      this.comment,
```

Comment:
This the `newSelectQuery` function has access to `this.comment`, we could potentially just reference it there rather than passing it in every time.

The advantage is much less boilerplate. Any downsides?